### ISSUE TITLE
incorporating NIH funded authors into permissions information

### ISSUE THREAD

**Issue by MichaelAEvans** [JATS4R/JATS4R-Participant-Hub#116]
Hello, can't make the call again today but have a question regarding NIH employees and licenses.

Our (F1000Res) permissions section is:

<permissions>
                <copyright-statement>Copyright: © 2014 Mirabello L et al.</copyright-statement>
                <copyright-year>2014</copyright-year>
                <license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/4.0/">
                    <license-p>This is an open access article distributed under the terms of the Creative Commons Attribution Licence, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</license-p>
                </license>
            </permissions>

We have a sentence we want to include for NIH/US Govt employees: "The author(s) is/are employees of the US Government and therefore domestic copyright protection in USA does not apply to this work. The work may be protected under the copyright laws of other jurisdictions when used in those jurisdictions."

I'm unsure of how to get this into the <permissions> info. Adding it to the end of the text in <license-p> would seem to violate the "It is the responsibility of the content producer to ensure that the human-readable version of the license statement matches the (machine-readable) license URI" rule as this line is not part of the CC BY 4.0 license. 

However, adding it as we have in the past as another full <license> within the <permissions> section would also be frowned upon wouldn't it, for having two separate licenses purporting to be the license for the article? This is the sort of tagging we've used previously:

<permissions>
                <copyright-statement>Copyright: © 2015 Evans M</copyright-statement>
                <copyright-year>2015</copyright-year>
                <license license-type="open-access" xlink:href="http://creativecommons.org/licenses/by/4.0/">
                    <license-p>This is an open access article distributed under the terms of the Creative Commons Attribution Licence, which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</license-p>
                </license>
                <license license-type="open-access">
                    <license-p>The author(s) is/are employees of the US Government and therefore domestic copyright protection in USA does not apply to this work. The work may be protected under the copyright laws of other jurisdictions when used in those jurisdictions.</license-p>
                </license>
            </permissions>

Thanks!

**Comment by Melissa37** [comment]
Hi Michael

They are covered by a different CC license (CCO)?

This is what we do:

`<license xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">
<license-p>
This is an open-access article, free of all copyright, and may be freely reproduced, distributed, transmitted, modified, built upon, or otherwise used by anyone for any lawful purpose. The work is made available under the
<ext-link ext-link-type="uri" xlink:href="http://creativecommons.org/publicdomain/zero/1.0/">Creative Commons CC0 public domain dedication</ext-link>.
</license-p>`

**Comment by MichaelAEvans** [comment]
Hi Melissa, thanks for the response.

I'm not sure that license is ideal - CC0 Public Domain waives all copyright worldwide, whereas the NIH only require it to be public domain in the USA. So the author would be waiving more rights than they need to.

Not sure how much that matters in reality? Any other publisher's able to share what they're doing for these?

Cheers,
Mike.

**Comment by Klortho** [comment]
> However, adding it as we have in the past as another full <license> within the <permissions> section would also be frowned upon wouldn't it, for having two separate licenses purporting to be the license for the article? 

I don't think that would be frowned upon at all.  It's something we haven't talked about. We've talked about multiple licenses for different parts of the document, and about licenses that change over time, but not about multiple licenses that apply to the whole document. IMO, that's the *right* way to do it, and there's nothing wrong with it. Any bot reading the articles could use the most permissive license that it finds.

I agree with you that adding the clause into the description of the CC-BY license element would be wrong, and would violate our rule about having the machine- and human- readable licenses match.  So, really, you have two options:

1. Change or remove the license URI, or
2. Add another \<license> element.

(1) would be bad, because then the open nature of the article would be completely lost to bots. So, I think (2) is the way to go.

**Comment by Melissa37** [comment]
Can you give an example of what you mean Chris as I am confused. The license needs to be a uri that is machine readable, so this sentence  would not cover it unless there is a specific URI for it? "The author(s) is/are employees of the US Government and therefore domestic copyright protection in USA does not apply to this work. The work may be protected under the copyright laws of other jurisdictions when used in those jurisdictions."

**Comment by Klortho** [comment]
What I mean is that I agree with Mike that adding that sentence to the CC-BY license element would be bad:

```xml
<permissions>
  <license xlink:href="http://creativecommons.org/licenses/by/4.0/">
    <license-p>This is an open access article distributed under the terms of the Creative 
      Commons Attribution Licence, which permits unrestricted use, distribution, and 
      reproduction in any medium, provided the original work is properly cited.
      The author(s) is/are employees of the US Government and therefore domestic 
      copyright protection in USA does not apply to this work. The work may be protected 
      under the copyright laws of other jurisdictions when used in those jurisdictions.
    </license-p>
  </license>
</permissions>
```

Because then the human-readable text doesn't match the machine-readable URI. So, I think either he should go with CC0, which was your suggestion (and seems simplest) or add another license element:

```xml
<permissions>
  <license xlink:href="http://creativecommons.org/licenses/by/4.0/">
    <license-p>This is an open access article distributed under the terms of the Creative 
      Commons Attribution Licence, which permits unrestricted use, distribution, and 
      reproduction in any medium, provided the original work is properly cited.</license-p>
  </license>
  <license>
    <license-p>The author(s) is/are employees of the US Government and therefore domestic 
      copyright protection in USA does not apply to this work. The work may be protected 
      under the copyright laws of other jurisdictions when used in those jurisdictions.
    </license-p>
  </license>
</permissions>
```

I don't know of any canonical URI for this US-government-public-domain, which is described here: https://www.usa.gov/government-works.

**Comment by MichaelAEvans** [comment]
Thanks for clarifying Chris. 

I was just nervous about adding a second license element within the permissions section as we've been asked recently to remove a second license element we had in there for our data. I understand now that is a different issue though.

My only concern with doing these with a CC0 license is my point that NIH only require the public domain in the US rather than worldwide as the license grants - I worry a clued up author wouldn't be happy to publish under that license. Anyway - that is something for me to talk to people here about, and it's good to know I have the extra <license-p> solution as a possibility if the legal people here advise against CC0.

Thanks!
